tests/suites/test_suite_psa_crypto.function

```diff
@@ -1155,6 +1155,25 @@ void mac_key_policy( int policy_usage_arg,
                                  mac, PSA_MAC_MAX_SIZE, &mac_len ),
                 expected_status_sign );
 
+    /* Calculate the MAC, multi-part case. */
+    PSA_ASSERT( psa_mac_abort( &operation ) );
+    status = psa_mac_sign_setup( &operation, key, exercise_alg );
+    if( status == PSA_SUCCESS )
+    {
+        status = psa_mac_update( &operation, input, 128 );
```

Comment:
When we want to test multi-part operation it should not be limited to only one call of `psa_mac_update()`. I think we should split the input into few parts. I suggest to test the following cases: 0, 1, 2, 3 parts.

Reply:
@mprse should I repeat the full multi-part with different chunk sizes, or split the update in multiple chunk of different sizes ?

Reply:
Maybe it would be good to split tests for one-shot and multi part. Then we would have one test case for single shot and few cases for multi-part (with different number of parts, and possibly also different sizes of parts).

Reply:
The `mac_key_policy` meant for test the key usage and policy settings and no need for separate multipart operation test cases. This kind of test should be added for tests which checks the validity of the mac operation: `mac_sign` and `mac verify`. For this the existing test cases can't be used, new ones must be created with proper output value.

Reply:
@gabor-mezei-arm I reused the same schema as AEAD multipart to implement the mac sign/verify with different chunk sizes

Reply:
I plan to rebase in order to remove the revert commit if we agree on what I pushed